.travis.yml

```diff
@@ -100,6 +100,26 @@ jobs:
         - NODE_VERSION: "10"
         - VARIANT: "stretch"
 
+    - stage: Build
```

Comment:
Think these changes need to be done in to https://github.com/nodejs/docker-node/blob/master/travis.yml.template first

Reply:
Aren't they?

Reply:
You're right, the individual versions are generated from the template, but that only needs to be updated if other parts of the .travis.yml are changed

Reply:
But v11.0.0 is whole new, not sure if I understand properly?

Reply:
Don't worry, your change is right. I was mixing it up with changes to the stages like the lints that need to be done in the template file